sui/token_bridge/sources/vaa.move

```diff
@@ -11,17 +11,26 @@
 /// in its `State`. If the encoded VAA passes through `parse_and_verify` again,
 /// it will abort.
 module token_bridge::vaa {
+    use wormhole::bytes32::{Bytes32};
     use wormhole::consumed_vaas::{Self};
+    use wormhole::external_address::{ExternalAddress};
     use wormhole::vaa::{Self, VAA};
 
     use token_bridge::state::{Self, State};
     use token_bridge::version_control::{Vaa as VaaControl};
 
-    // All friends need `parse_and_verify`.
     friend token_bridge::create_wrapped;
     friend token_bridge::complete_transfer;
     friend token_bridge::complete_transfer_with_payload;
 
+    struct TokenBridgeMessage {
```

Comment:
could you document the assumptions about this type here? so they're easy to review + ensure that future changes don't violate them